src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -27,18 +17,15 @@
     <Reference Include="Microsoft.Extensions.Caching.Memory" />
     <Reference Include="Microsoft.Extensions.FileProviders.Composite" />
     <Reference Include="Microsoft.Extensions.FileProviders.Embedded" />
-
-    <!--
-      Temporary until we move the prerendering APIs into this assembly.
-    -->
-    <Reference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" />
```

Comment:
@javiercn I tried removing this and it worked. So I'm guessing this was left over from the pre-rendering work.